packages/waku/src/lib/vite-plugins/main.ts

```diff
@@ -17,6 +17,11 @@ const __dirname = fileURLToPath(new URL('.', import.meta.url));
 export function mainPlugin(config: Required<Config>): Plugin {
   return {
     name: 'waku:vite-plugins:main',
+    resolveId(source) {
+      if (source === 'cloudflare:workers') {
+        return { id: source, external: true };
+      }
+    },
```

Comment:
Nice. Though it's very small, I think I would like to have it a separate plugin. `packages/waku/src/lib/vite-plugins/cloudflare.ts` or something.